### ISSUE TITLE
Suggestion - enhanced profile details.

### ISSUE THREAD

**Issue by imcompiling** [Wingysam/Christmas-Community#114]
I think a good feature would be slightly enhanced profile details. It would be nice to be able to show things like shirt size, pants size, ring size in one place that doesn't have to be added when an item is added/deleted.

**Comment by Wingysam** [comment]
That's genius! I'll implement this as soon as I can.

**Comment by juzim** [comment]
Something I always wanted was a box for unspecific wishes like a profile free text field for "things that always make me happy" (flowers, Pokémon cards, theatre tickets etc).

**Comment by Wingysam** [comment]
I'm realizing that this issue should have been closed when I implemented the feature. So you're requesting the ability to add arbitrary fields to the info table on your wishlist instead of the preset ones in the app?

**Comment by juzim** [comment]
No, I think a single text field would be enough but ideally it would be shown above the wishlist not in the profile.

Of course my kids can add 5x Pokémon cards every year but having a short text above the list for stuff like that would be ideal.

I will think about it and create a new issue if necessary.
